src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+log = .
+log4j.rootLogger = INFO, LAMBDA
+
+# Define the LAMBDA Appender
+log4j.appender.LAMBDA=com.amazonaws.services.lambda.runtime.log4j.LambdaAppender
```

Comment:
Yep, this will log to CloudWatch: http://docs.aws.amazon.com/lambda/latest/dg/java-logging.html#java-wt-logging-using-log4j